.wwhrd.yml

```diff
@@ -42,3 +42,5 @@ exceptions:
 additional:
   # list here paths to additional licenses
   golang/go: "raw.githubusercontent.com/golang/go/master/LICENSE"
+  # tailscale/tailscale: "raw.githubusercontent.com/tailscale/tailscale/main/LICENSE"
```

Comment:
Is the comment intended ?

Reply:
Nope, I forgot to take it out. Thanks for finding -- just took it out.